### ISSUE TITLE
ipsec containers are not starting correctly on Ubuntu 14.04 with Docker 1.10

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#9191]
**Rancher versions:**
rancher/server: v1.6.3-rc3

**Infrastructure Stack versions:**
healthcheck: 0.3.1
ipsec: rancher/net:v0.11.3
network-services: rancher/network-manager:v0.7.3 rancher/metadata:v0.9.2
scheduler: rancher/scheduler:v0.8.1

**Docker version: (`docker version`,`docker info` preferred)**
1.10
**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
ubuntu 14.04
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
GCE
**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node rancher
**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
cattle
**Steps to Reproduce:**
- Start Rancher server and register two nodes

**Results:**

Ipsec fails to start with the following error:
```
Expected state running but got error: Couldn't bring up network: failed to find plugin "rancher-bridge" in path [/opt/cni/bin /var/lib/cni/bin /usr/local/sbin /usr/sbin /sbin /usr/local/bin /usr/bin /bin]
```

**Comment by leodotcloud** [comment]
Looking at the logs, `network-manager` couldn't communicate with `rancher-metadata` and `rancher-metadata` couldn't communicate with rancher/server as it's `eth0` went missing. This is due to some problem figuring out the right index of veth by `vethsync` component inside `network-manager`.

**Comment by leodotcloud** [comment]
The current approach of entering the namespace and trying to read the ParentIndex of the interface doesn't seem to work reliably on all the cloud providers and OS versions. Hence removing oldvethsync.

**Comment by leodotcloud** [comment]
Fixed with `network-manager: v0.7.4`.

**Comment by galal-hussein** [comment]
Rancher Version: v1.6.3-rc4

Can't see the problem any more on any OS on any cloud provider.
